src/storage/hummock_test/src/snapshot_tests.rs

```diff
@@ -116,7 +114,6 @@ async fn test_snapshot_inner(enable_sync: bool, enable_commit: bool) {
                 .commit_epoch(epoch1, ssts)
                 .await
                 .unwrap();
-            vm.refresh_version(mock_hummock_meta_client.as_ref()).await;
```

Comment:
How to ensure the latest version is pushed? Same below.

Reply:
> How to ensure the latest version is pushed? Same below.

fixed